packages/common/aws.js

```diff
@@ -299,8 +309,21 @@ exports.s3PutObjectTagging = (bucket, key, tagging) =>
 * @param {string} key - key for object (filepath + filename)
 * @returns {Promise} - returns response from `S3.getObject` as a promise
 **/
-exports.getS3Object = (bucket, key) =>
-  exports.s3().getObject({ Bucket: bucket, Key: key }).promise();
+exports.getS3Object = async (bucket, key) => {
```

Comment:
If there is an error in an S3 API call, the stack trace gives no information about what the failing operation was or where it occurred.  This code provides a more useful stack trace when someone calls `getS3Object()`, which actually shows what the call was, where it was, and what the params were.  Makes debugging a _lot_ easier.